.gitignore

```diff
@@ -32,5 +38,42 @@ scripts/coverage
 
 # typescript
 packages/*/*.tsbuildinfo
+apps/*/*.tsbuildinfo
 
+# Logs
 *storybook.log
+
+# Linux system files
+*~
+
+# Android/Java
+*.keystore
+**/android/.gradle/
+**/android/app/build/
+**/android/app/.cxx/
+**/android/local.properties
+**/android/.idea/
+**/android/.gradle/
+**/android/build/
+
+# iOS/Swift
+**/ios/Pods/
+**/ios/build/
+**/ios/DerivedData/
+**/ios/.xcuserdata/
+**/ios/.xcode.env.local
+*.mode1v3
+*.pbxuser
+*.xccheckout
+*.xcuserstate
+*.xcworkspace
+!ios/Podfile.lock
+!ios/Pods/
```

Comment:
Don't we typically want to ignore `Pods` files? See mobile https://github.com/BranchMetrics/ios-branch-deep-linking-attribution/blob/615267ea0b44bc3be657b84fbc9ff7c86688fa58/.gitignore#L16 @Cal-L could you please check these and what is appropriate to ignore?

Reply:
Good catch. Pods should be ignored as well